pkg/version/version.go

```diff
@@ -0,0 +1,43 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package version
+
+import (
+	"fmt"
+
+	"github.com/blang/semver"
+)
+
+var criContainerdVersion = "UNKNOWN"
+
+func validateSemver(sv string) error {
+	badVersion, err := semver.Parse(sv)
+	if err != nil {
+		return fmt.Errorf("couldn't parse kubernetes version %q: %v", badVersion, err)
```

Comment:
I can't discard badVersion since semver.Parse returns two variables. There is a [semver.Validate](https://godoc.org/github.com/blang/semver#Version.Validate), but it require passing a semver.Version variable.